hasura/metadata/databases/default/tables/public_user.yaml

```diff
@@ -67,11 +70,3 @@ update_permissions:
         id:
           _eq: X-Hasura-User-Id
       check: null
-delete_permissions:
```

Comment:
ok for now but let's please note internally we need to add/confirm full support for: deleting your own user account; orphaned users who don't have a team (frontend handling mainly).

Reply:
@pdtfh First we need to discuss how it will look like on the UI side. Because now we will have two different processes:
1. Removing user from the team (Deleting `membership` table row)
2. Deleting user from the Dev Portal completely (Deleting `user` table row)

Reply:
👍 
1. Should be from the list of teammates.
2. Should only be done by each person, can be done from profile settings. We also need to handle the case of you logging in and not belonging to any team. You should be able to create a new team or delete your account. (CC @andy-t-wang to make sure it's covered in design).

Reply:
> Should only be done by each person

Updated. Add check `{id: {_eq: X-Hasura-User-Id}}`